corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -128,7 +128,7 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
           </div>
         </div>
         <!-- /ko -->
-        <span data-bind="html: ko.utils.unwrapObservable($data.caption_markdown) || caption()"></span>
+        <span class = "webapp-markdown-output" data-bind="html: ko.utils.unwrapObservable($data.caption_markdown) || caption()"></span>
```

Comment:
There are a couple of other places in this file that use `caption_markdown` - one for groups, the other I think is help text - that should also get this class.